packages/slice-machine/pages/_app.tsx

```diff
@@ -171,8 +172,4 @@ function MyApp({
   );
 }
 
-MyApp.getInitialProps = async (appContext: AppContext) => {
-  return await App.getInitialProps(appContext);
-};
-
-export default MyApp;
+export default dynamic(() => Promise.resolve(App), { ssr: false });
```

Comment:
Just double checking: is `Promise.resolve()` necessary?

Reply:
Yes, the second one looks good